CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v7.0 (in development) ###
+
+* Replaces pg_dist_shard_placement metadata table with pg_dist_placement
```

Comment:
I wasn't sure what to do here, I can remove this change but it seems preferable for commits to update the changelog when they touch something user facing? Having a useful git blame sounds nice this makes it less likely we'll forget to add something to the log when doing the release.